vortex-array/src/array/varbin/stats.rs

```diff
@@ -83,7 +80,9 @@ pub fn compute_varbin_statistics<T: ArrayAccessor<[u8]> + Deref<Target = Array>>
         | Stat::TrueCount
         | Stat::RunCount
         | Stat::BitWidthFreq
-        | Stat::TrailingZeroFreq => {
+        | Stat::TrailingZeroFreq
+        | Stat::Min
```

Comment:
Is this true now? Why would min/max never get to this point? Don't you need to call into the min/max compute fn?

Reply:
That is done in `pub fn compute_stats`